### DISCUSSION TITLE
Query string that has accents

### DISCUSSION THREAD

**Discussion by JafetGuerra** [ClickHouse/ClickHouse#43631] [Q&A] [answered]
How to query a string that has portuguese accents in clickhouse without having to considering it?

Example:
    SELECT * FROM table
    WHERE positionCaseInsensitive(name, 'that') > 0
    
    Should return any rows that has 'that', 'thât', 'thát' so on.

It is possible to do so in clickhouse?

**Comment by mo-avatar** [comment]
I don't think it is possible, at least for now,  because they are totally different when encoding. The only possible way I can come out  to solve this problem is to have a vocalbulary dictionary in clickhouse and using dictGet method. Any way it may requires some kind of linguistic information.

**Reply by mo-avatar** [reply, marked as answer]
I guess may be there is one way  which you can do it, that is write your self a excutable script style of udf to make an character transformation then apply positionCaseInsensitive on it .
https://presentations.clickhouse.com/meetup55/udf_in_clickhouse/

**Reply by JafetGuerra** [reply]
Hmm thanks for your answer! I thought that it wouldn't be possible by default. I will look into this presentation.

Thanks!!
